org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/UiSession.java

```diff
@@ -605,6 +611,52 @@ public void dispose() {
     m_sessionMetrics.sessionDestroyed(SESSION_TYPE);
   }
 
+  protected void runInModelJob(Runnable r, String name) {
+    runInModelJob(r, name, false);
+  }
+
+  protected void runInModelJob(Runnable r, String name, boolean wait) {
+    if (r == null) {
+      return;
+    }
+
+    // current thread is the model thread -> run
+    if (ModelJobs.isModelThread()) {
+      r.run();
+      return;
+    }
+
+    // current thread is NOT the model thread and the client session is active -> run in model job
+    final IClientSession clientSession = getClientSession();
+    if (clientSession != null && clientSession.isActive() && !clientSession.isStopping()) {
+      final ClientRunContext clientRunContext = ClientRunContexts.copyCurrent(true).withSession(clientSession, true);
+      IFuture<Void> modelJob = ModelJobs.schedule(
+          r::run,
+          ModelJobs.newInput(clientRunContext)
+              .withName(name)
+              .withExceptionHandling(null, false));// Propagate exception to caller (UIServlet)
```

Comment:
Missing space between code and comment.

Is the comment still accurate? Is it always the UiServlet? If yes, a lowercase "i" should be used. If not, maybe it is better to remove the "(UIServlet)" part entirely.